src/utils/texture.js

```diff
@@ -94,9 +94,9 @@ function loadVideo (material, data, src) {
 
   // Cache as promise to be consistent with image texture caching.
   textureCache[calculateVideoCacheHash(videoEl)] = Promise.resolve(texture, videoEl);
-  handleVideoTextureLoaded(texture, videoEl);
+  handleVideoTextureLoaded(texture);
 
-  function handleVideoTextureLoaded (texture, videoEl) {
```

Comment:
I think we need this parameter for when the promise resolves. It might just be a matter of renaming this var.